site/gatsby-site/src/utils/AlgoliaUpdater.js

```diff
@@ -112,8 +120,16 @@ const reportToEntry = ({ incident = null, report }) => {
     featured: featuredValue,
     flag: report.flag,
     is_incident_report: report.is_incident_report,
+    namespaces: classifications.map((c) => {
+      return Object.keys(c.tree)[0];
```

Comment:
is it fair to assume the keys of the `classification.tree` are ordered here?

Reply:
It is more that the order doesn't matter.